helpers/or.js

```diff
@@ -9,21 +9,12 @@ const _ = require('lodash');
  * {{#or 1 0 0 0 0 0}} ... {{/or}}
  */
 const factory = () => {
-    return function() {
-        var args = [],
-            opts,
-            any;
+    return function(...args) {
 
-        // Translate arguments to array safely
-        for (var i = 0; i < arguments.length; i++) {
-            args.push(arguments[i]);
-        }
-
-        // Take the last argument (content) out of testing array
-        opts = args.pop();
+        // Take the last arg out of args array
+        const opts = args.pop();
 
-        // Without options hash, we check all the arguments
-        any = _.any(args, function (arg) {
+        const any = _.any(args, function (arg) {
```

Comment:
Could we add a comment here like the one that is removed to describe what we are doing here?

Reply:
fixed.